core/src/main/java/com/yahoo/ycsb/measurements/OneMeasurementHistogram.java

```diff
@@ -36,11 +36,39 @@
 	public static final String BUCKETS="histogram.buckets";
 	public static final String BUCKETS_DEFAULT="1000";
 
+
+	/**
+	 * Specify the range of latencies to track in the histogram.
+	 */
 	int _buckets;
+
+	/**
+     * Groups operations in discrete blocks of 1ms width.
+     */
 	int[] histogram;
+	
+	/**
+	 * Counts all operations outside the histogram's range.
+	 */
 	int histogramoverflow;
+	
+	/**
+	 * The total number of reported operations.
+	 */
 	int operations;
-	long totallatency;
+	
+	/**
+	 * The sum of each latency measurement over all operations.
+	 * Calculated in ms.
+	 */
+	double totallatency;

```

Comment:
let's leave it our intermediate calculations using fixed point then. I presume the double for display is an attempt to get fractional units. I'd rather not deal with floating point calculation error in our core measurements.